views/partials/beerpartial.hbs

```diff
@@ -0,0 +1,5 @@
+<img src='{{image_url}}' alt='{{name}}'>
+
+{{name}}
+{{description}}
+{{tagline}}
```

Comment:
You forgot to wrap the content in the respective tags: <p>{{name}}<p> etc.  This way only some of the text appears but all of it plus it's not actually added to the DOM (try to inspect it).